scipy/stats/qmc.py

```diff
@@ -0,0 +1,648 @@
+"""Quasi-Monte Carlo methods.
+
+Define function to generate sample of points in the unit hypercube.
+
+"""
+
+from __future__ import division
+
+import copy
+import numpy as np
+from scipy.optimize import brute
+from scipy._lib._util import check_random_state
+
```

Comment:
You want to add an `__all__` list here with the functions that are public

Reply:
Will do.